pages/spacedeck/[[...roomId]].js

```diff
@@ -0,0 +1,297 @@
+        }
+        if (!remove.ok) {
+        // @TODO callback function to give user feedback when removing on the server fails
+            setRemovingLink(false);
+
+            return;
+        }
+        await auth.getAuthenticationProvider('matrix').removeSpaceChild(serviceSpaceId, roomId);
+        await matrix.leaveRoom(roomId);
+        router.push('/sketch');
+        setRemovingLink(false);
+    };
+
+    const ActionNewSketch = ({ callbackDone }) => {
+        const [sketchName, setSketchName] = useState('');
+        const [loading, setLoading] = useState(false);
+
+        const createNewSketchRoom = async () => {
+            setLoading(true);
+
+            const create = await spacedeck.createSpace(sketchName);
+            const link = getConfig().publicRuntimeConfig.authProviders.spacedeck.baseUrl + '/spaces/' + create._id;
+            const roomId = await createSketchRoom(link, sketchName);
+            router.push(`/sketch/${roomId}`);
+
+            callbackDone && callbackDone();
+            setLoading(false);
+        };
+
+        return (
+            <Form onSubmit={(e) => { e.preventDefault(); createNewSketchRoom(); }}>
+                <input type="text" placeholder={t('sketch name')} value={sketchName} onChange={(e) => setSketchName(e.target.value)} />
+                <button type="submit" disabled={!sketchName || loading}>{ loading ? <LoadingSpinnerInline inverted /> : t('Create sketch') }</button>
+                { errorMessage && <ErrorMessage>{ errorMessage }</ErrorMessage> }
+            </Form>);
+    };
+
+    const ActionExistingSketch = ({ callbackDone }) => {
+        const [sketchName, setSketchName] = useState('');
+        const [sketchLink, setSketchLink] = useState('');
+        const [validLink, setValidLink] = useState(false);
+        const [loading, setLoading] = useState(false);
+
+        const handleExistingSketch = (e) => {
+            // we check if the link is valid for the service (has the same base url)
+            if (e.target.value.includes(getConfig().publicRuntimeConfig.authProviders.spacedeck.baseUrl)) setValidLink(true);
+            else setValidLink(false);
+            setSketchLink(e.target.value);
+        };
+
+        const handleSubmit = async (e) => {
+            setLoading(true);
+            e.preventDefault();
+            await createSketchRoom(sketchLink, sketchName);
+            callbackDone && callbackDone();
+            setLoading(false);
+        };
+
+        return (
+            <Form onSubmit={handleSubmit}>
+                <input type="text" placeholder={t('sketch name')} value={sketchName} onChange={(e) => setSketchName(e.target.value)} />
+                <input type="text" placeholder={t('link to sketch')} value={sketchLink} onChange={handleExistingSketch} />
+                { !validLink && sketchLink !=='' && <ErrorMessage>{ t('Make sure your link includes') }:  { getConfig().publicRuntimeConfig.authProviders.spacedeck.baseUrl }</ErrorMessage> }
```

Comment:
Can we reuse the `Make sure your link includes "{{url}}"` error message from the /write PR?
And therefore move the translation of that to the `_defaults.json` file?